src/style.css

```
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
+.page-content {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+  max-width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  align-self: center;
+
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  margin: 0 0 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 0 0 16px;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 9px;
+}
+
+.stars__star {
+  display: flex;
+  background: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background: url(images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__reviews {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-word {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+  color: #616070;
+}
+
+.card__price-amount {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  cursor: pointer;
+  border: 1px solid #00acdc;
```

You can add here transparent color and change it on hover
```suggestion
  border: 1px solid transparent;
```